lib/maybe_date.rb

```diff
@@ -13,12 +13,20 @@ def coerce(value)
     if value.is_a?(String)
       Date.parse(value)
     elsif value.respond_to?(:values_at)
-      ymd = value.values_at(:year, :month, :day).map(&:to_i)
```

Comment:
Can we add specs to this? Also, if this is the same as in pvb2 we should extract it out into a gem